### PR TITLE
Calculator Created

### PR DESCRIPTION
**Pull request by Abdulwhbhussain**
Description
In this project, you should continue with the development of the Math Magicians app. You will develop a React component that will hold the core functionality: a calculator.

General requirements
Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
Make sure that you used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
JavaScript requirements
Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
Project requirements
In the previous project, you worked on the setup of a React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html), now you should:

Delete all the boilerplate from Create React App (text, images, styles).
Create a new directory called components.
Inside components, create a new Calculator.js file.
In Calculator.js, create a React component that matches this design:
Calculator

Implement the design only (JSX and CSS)
Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
Use functional components
Create at least 1 child component
Once the Calculator is ready, import it and use it in your top-level component: App.js
Test it in the browser; it should render without problems.
NOTE: Don't add any extra design. Don't add any logic.

### CODE REVIEW

File: src/components/Calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    font-size: 3rem;
+    font-weight: bold;
+    background-color: #A9A9A9;
+    color: #ffffff;
+    margin-top: 0;
+    margin-bottom: 0;
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+  }
```
Comment (AlphaNtihinduka) [inline src/components/Calculator.jsx]: - [x] Please consider reducing the width of the calculator design to ensure that the design is close the wireframe. This is to ensure that the calculator has a good user interface for easy use.
- [x] [OPTIONAL] You may consider centering the calculator to have a better and professional design ☺️.

File: src/components/Calculator.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <ButtonCalculator className="five" name="5" />
+    <ButtonCalculator className="six" name="6" />
+    <ButtonCalculator className="minus" name="-" variant="orange" />
+    <ButtonCalculator className="one" name="1" />
+    <ButtonCalculator className="two" name="2" />
+    <ButtonCalculator className="three" name="3" />
+    <ButtonCalculator className="plus" name="+" variant="orange" />
+    <ButtonCalculator className="zero" name="0" />
+    <ButtonCalculator className="dot" name="." />
+    <ButtonCalculator className="equal" name="=" variant="orange" />
```
Comment (AlphaNtihinduka) [inline src/components/Calculator.jsx]: - [ ] Please ensure that these calculator buttons match the wireframe design. If you notice, `0` covers 50% width of the calculator design. Kindly apply those changes to have a professional calculator design ☺️ 

Your design 😎 
image

Wireframe design 😎 
image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @Abdulwhbhussain 🙋‍♂️
Good job working calculator components. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- Used styled components ✔️ 
- Proper use of functional components ✔️
- Proper function importation ✔️
- Used arrow functions ✔️ 
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- _Kindly consider the inline review comments ☺️_ 

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: src/components/Calculator.jsx (hunk 1 above)
Comment (Abdulwhbhussain) [reply src/components/Calculator.jsx]: Sure Bro! I am Working on it.

File: src/components/Calculator.jsx (hunk 2 above)
Comment (Abdulwhbhussain) [reply src/components/Calculator.jsx]: Updated Bro!

**Review by mikethreels — APPROVED** [review]
Hi Wahab, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job following the instructions of the previous CR.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — COMMENTED** [review]
Hi @Abdulwhbhussain, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- All required changes fixed ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**